datadriven/src/sgpp/datadriven/datamining/modules/visualization/VisualizerClassification.cpp

```diff
@@ -846,7 +846,7 @@ void VisualizerClassification::storeHeatmapJsonClassification(DataMatrix &matrix
     jsonOutput["data"][graphIndex].addDictAttr("marker");
     jsonOutput["data"][graphIndex]["marker"].addIDAttr("symbol", "\"star-dot\"");
     jsonOutput["data"][graphIndex]["marker"].addIDAttr("size",
-      static_cast<size_t>(10));
+      static_cast<u_int64_t>(10));
```

Comment:
```suggestion
      static_cast<uint64_t>(10));
```
The tests picked up a small typo!